internal/pkg/skuba/deployments/ssh/kernel.go

```diff
@@ -43,10 +47,25 @@ var (
 )
 
 func init() {
+	stateMap["kernel.check-modules"] = kernelCheckModules
 	stateMap["kernel.load-modules"] = kernelLoadModules
 	stateMap["kernel.configure-parameters"] = kernelConfigureParameters
 }
 
+func kernelCheckModules(t *Target, data interface{}) error {
+	var exist = true
+	for _, module := range modules {
+		if err := infoModule(t, module); err != nil {
+			exist = false
+		}
+	}
+	if !exist {
+		_, _, err := t.ssh("zypper", "install", "--no-confirm", "--", "kernel-default", "-kernel-default-base")
```

Comment:
I'm not sure about this change, I would prefer that `skuba` does not do this. This will require a reboot for this to be effective and I think it belongs to the terraform states; mainly because at some point we have the expectation that `kernel-default-base` will be enough. We just ran out of time for `kernel-default-base` on SP1 to include everything we needed for cilium.

I would just perform the modules check from skuba, and if some modules are missing, we error out with a clear message stating the problem, so the user can fix it on their own.

Reply:
Oh, OK, so maybe I misunderstood the description of the Issue, I though that we wanted to remove this from `terraform`.

So only having the module checks and ensure the `sysctls` checked should be enough then.

Reply:
>So only having the module checks and ensure the sysctls checked should be enough then.

Yes.

And I think that we should provide VM images for CaaSP with `kernel-default` as a long term solution.